cmd/webhook/main.go

```diff
@@ -111,12 +111,19 @@ func SharedMain(handlers map[schema.GroupVersionKind]webhook.GenericCRD, factori
 		SecretName:     "webhook-certs",
 		WebhookName:    fmt.Sprintf("webhook.%s.events.cloud.run", system.Namespace()),
 	}
+
+	stats, err := webhook.NewStatsReporter()
```

Comment:
This was broken by https://github.com/knative/pkg/issues/517

Reply:
Should be a separate PR